server/db/schema.rb

```diff
@@ -10,13 +10,15 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2022_10_05_160613) do
+
+ActiveRecord::Schema[7.0].define(version: 2022_10_06_043825) do
   create_table "users", force: :cascade do |t|
     t.string "username"
     t.string "email"
     t.string "password_digest"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string "usertype"
```

Comment:
maybe we can have a default of `student`s?

Reply:
please specify more

Reply:
Please see other lines in this file that has `, default:`